deps/verifier/src/snp/mod.rs

```diff
@@ -34,43 +35,43 @@ const LOADER_SPL_OID: Oid<'static> = oid!(1.3.6 .1 .4 .1 .3704 .1 .3 .1);
 
 #[derive(Debug)]
 pub struct Snp {
-    vendor_certs: VendorCertificates,
+    vendor_certs: Chain,
 }
 
-pub(crate) fn load_milan_cert_chain() -> &'static Result<VendorCertificates> {
-    static MILAN_CERT_CHAIN: OnceLock<Result<VendorCertificates>> = OnceLock::new();
-    MILAN_CERT_CHAIN.get_or_init(|| {
-        let certs = X509::stack_from_pem(include_bytes!("milan_ask_ark_asvk.pem"))?;
-        if certs.len() != 3 {
-            bail!("Malformed Milan ASK/ARK/ASVK");
+fn read_certs_from_file() -> Vec<X509> {
```

Comment:
Not sure of the value of splitting this into two functions. This new one should still refer to `milan` since it is only loading the milan certs. Also this isn't exactly reading them from a file.

Reply:
I am splitting this into two functions because the former behavior of utilizing a `OnceLock` around a `Result` leaves room for unexpected behavior, as it returns a reference to a `Result`. I suppose I could re-implement this using a `lazy_static!` to ensure it is only allocated once, and then use a Mutex to make sure access is serial. 

Ultimately, I was just trying to get rid of the UB.

Reply:
Renamed. Let me know what you think about the latest commit.